source/common/tools/student-virtual-coach.rst

```diff
@@ -0,0 +1,54 @@
+.. meta::
+   :description: Virtual Coach is a quick AI solution for helping students with simple queries.
+
+
+.. _student-virtual-coach:
+
+Virtual Coach 
+=============
+
+Virtual Coach provides students the ability to use AI to help you with programming tasks without receiving answers. This will give you an quick solution/hint which can reduce the possibility of getting stuck and have more productive and positive learning experiences. It also reduce the hassle to reach out to TAs and teachers for the simple questions.
```

Comment:
It also reduce the hassle to reach out to TAs and teachers for the simple questions. > It can also reduce the need for you to have to reach out to TAs and teachers for the simple questions.